lib/pubid/ieee/identifier.rb

```diff
@@ -37,5 +53,15 @@ def type
     def year
       "-#{@year}" if @year
     end
+
+    def alternative
+      if @alternative
+        if @alternative.is_a?(Array)
+          " (#{@alternative.map { |a| Identifier.new(**a) }.join(', ')})"
```

Comment:
Does this step know what kind of Identifier the alternative identifier is? e.g. ISO, IEC, ASA ...